src/framework/link/Link.jsx

```diff
@@ -28,6 +34,7 @@ Link.propTypes = {
   href: PropTypes.string,
   onClick: PropTypes.func,
   data: PropTypes.any,
+  isTargetBlank: PropTypes.bool,
```

Comment:
I think we should leverage convention for components which are simple wrappers around HTML elements. So I think we should just make this `target` instead, and require the dev to pass in `"_blank"`.